app/models/miq_ae_method.rb

```diff
@@ -17,7 +17,7 @@ class MiqAeMethod < ApplicationRecord
 
   AVAILABLE_LANGUAGES  = ["ruby", "perl"]  # someday, add sh, perl, python, tcl and any other scripting language
   validates_inclusion_of  :language,  :in => AVAILABLE_LANGUAGES
-  AVAILABLE_LOCATIONS = %w(builtin inline expression playbook).freeze
+  AVAILABLE_LOCATIONS = %w(builtin inline expression playbook ansibletemplate).freeze
```

Comment:
naming is hard. The full name is "ansible tower job template" which is too long. How about just jobtemplate? can we have hyphen or underscore? must it be all lower case?

Reply:
@mkanoor @gmcculloug What do you think about @bzwei's suggestion of the method name?

Reply:
@lfu is this Job Template or Workflow? I think we should say Ansible Job template if its the job template, if its the Workflow we should use Ansible Workflow.

Reply:
@lfu  the internal implementation could be the same but to the user using the product they are 2 different things correct? If you went to the Ansible UI do they collapse the Job template and the Workflow into a single entity or do they show them differently. What happens if they have a Job Template and Workflow with the same name, would they be listed separately or is it distinct on name?

Reply:
They are listed differently.
<img width="792" alt="screen shot 2019-01-22 at 10 18 41" src="https://user-images.githubusercontent.com/2593270/51545019-2126d500-1e2f-11e9-81db-0dc901acd904.png">

Reply:
@bzwei @mkanoor This method works for both job template and workflow template. And it is an internal automate method name. I'd like to get a final name that we all agree on before I make the change.

Reply:
@lfu @tinaafitz 
I think those names are displayed to the end user, here Is a screenshot
![screen shot 2019-01-22 at 11 47 47 am](https://user-images.githubusercontent.com/6452699/51551130-ce9fe580-1e3b-11e9-8064-f331ef34e089.png)

Till now we have had single words now we have 2 or more words "Job Template" or "Ansible Workflow", so maybe for now we could just use job_template and workflow

Reply:
@gmcculloug Since Ansible treats the Job Template and the Workflow separately it would be nice if we treat them separately, the internal implementation could be the same. That also avoids the confusion when an end user or customer looks at the Method type, they would know right away that we support both Job Template and Workflow, otherwise it gets lost and if there are long names the (workflow) suffix would not be visible.

Reply:
I think it is inaccurate to say they are treated separately in Ansible.  They appear in the same list-view called "Templates" and the only difference is the Type field.  Otherwise from a user standpoint there is nothing different.

I would hope it is an exception case where an admin is creating the same named objects as both JobTemplate and Workflow, but you never know.

![image](https://user-images.githubusercontent.com/2591569/51568795-92cf4500-1e68-11e9-8e47-13f81bc02411.png)

In Services we added Workflow to the same drop-down but I recall there is a separator between them.  I don't think that would be as easy to do here.

@lfu What's the effort to split them and display differently?

Reply:
@gmcculloug It can be easily handled by two different names. How about ```job_template``` and ```workflow_template```?

Reply:
I would suggest keeping `ansible` in the name to avoid any confusion, otherwise I'm ok with it.